README.md

```diff
@@ -77,11 +77,15 @@ output 138GB.
 These Luigi pipelines are used to preprocess the evaluation tasks
 into a common format for downstream evaluation.
 
-To run the preprocessing pipeline for all available tasks, with all available modes for each task:
+To run the preprocessing pipeline for all available tasks, with all
+available modes for each task:
 ```
 python3 -m hearpreprocess.runner all --mode all
 ```
 
+`all` could instead be a specific single task, or specific multiple
+tasks.
```

Comment:
Not sure what you mean by this.

Reply:
Oh, I understand. You mean all (the positional arg) could be replaced by a specific task, or a list of tasks. It's a bit confusing the way it is worded currently

Reply:
Updated the docs